docs/_docs/02-ug-configuration.md

```diff
@@ -2,7 +2,12 @@
 title: "Configuration"
 permalink: /docs/ug-configuration/
 excerpt: "How to configure Hyperspace for your needs."
-last_modified_at: 2020-06-20
+last_modified_at: 2020-06-23
 toc: true
 ---
-
+| Property name                                        | Default                                                                                          | Meaning                                                                                               | Since Version |
+|------------------------------------------------------|--------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------|---------------|
+| spark.hyperspace.system.path                         | "root/spark-warehouse/indexes"                                                                   | Folder to store Hyperspace index files.                                                     | 0.1.0         |
```

Comment:
The default should be "(value of `spark.sql.warehouse.dir`)/indexes" ? Not sure how to phrase this :)

Reply:
Also how about "Root directory to store Hyperspace index files"?

Reply:
Done.

Reply:
Looks like they use "value of", thus I suggested above.

Reply:
Fixed.